src/MyTested.AspNetCore.Mvc.Authentication/ComponentBuilderAuthenticationWithExtensions.cs

```diff
@@ -145,7 +145,7 @@ public static TBuilder WithUser<TBuilder>(this IBaseTestBuilderWithComponentBuil
         {
             var actualBuilder = (BaseTestBuilderWithComponentBuilder<TBuilder>)builder;
 
-            actualBuilder.HttpContext.User = ClaimsPrincipalBuilder.DefaultAuthenticated;
+            actualBuilder.HttpContext.User = BaseClaimsPrincipalUserBuilder.DefaultAuthenticated;
```

Comment:
Is there a reason for the `DefaultAuthenticated` property to be in the Base class?

Reply:
Because it is called from both WithClaimsPrincipalBuilder and WithoutClaimsPrincipalBuilder, probably would be better to be called directly by the class with inherits from the base, instead of using the base directly.